web/router.ex

```diff
@@ -61,6 +61,7 @@ defmodule CodeCorps.Router do
     resources "/user-roles", UserRoleController, only: [:create, :delete]
     resources "/user-skills", UserSkillController, only: [:index, :show, :create, :delete]
     resources "/role-skills", RoleSkillController, only: [:index, :show, :create, :delete]
+    resources "/project-skills", ProjectSkillController, only: [:index, :show, :create, :delete]
```

Comment:
Would reorder this alpha by resources, and cleanup other things while you're in there (like `/role-skills`).